apps/dashboard/app/javascript/dynamic_forms.js

```diff
@@ -508,6 +508,9 @@ function toggleMinMax(event, changeId, otherId) {
   [ 'max', 'min' ].forEach((dim) => {
     if(mm && mm[dim] !== undefined) {
       changeElement.attr(dim, mm[dim]);
+    } else {
+      const defaultVal = dim == 'min' ? 1 : 999;
+      changeElement.attr(dim, defaultVal);
```

Comment:
I love the precision here in making just a small enough change, but sadly, I feel like it's not enough.

It seems to me that we should embed the min or max to fall back on.  Take this for example, we should fall back to these values if they're defined. 

```yaml
some_element:
  widget: number_field
  min: 2
  max: 4
```

If we can't fallback to any values, then we just don't. I think the conservative route of just doing nothing is better than choosing 1 or 999. But we can document that these are fallback values when you use data-min or data-max sparingly.